src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -234,6 +240,7 @@ public ServicePoint getCheckoutServicePoint() {
 
   public Loan renew(DateTime dueDate, String basedUponLoanPolicyId) {
     changeAction("renewed");
+    removeActionComment();
```

Comment:
Do we want to remove the action comment in renewals? Expected behavior stated in CIRC-251 is

> Comment from renewal override displays under Comments for the row where the action reads Renewed through override

Is this method used for override as well?

Reply:
Seems like Marc gave a detailed explanation in the comment below.

Reply:
@dmytrotur ok